pkg/gitapi/gitapi.go

```diff
@@ -41,16 +40,16 @@ func GetList(config *config.Config) []string {
 
 	i := 0
 	var repoNames []string
-	logr.Println("[GitAPI] Found Repositories:")
 	for _, repo := range repos {
 		i++
 		name, ok := repo["name"].(string)
 		if !ok {
 			logr.Errorf("[GitAPI] failed converting the repository name to string: %v\n", err)
 		} else {
 			repoNames = append(repoNames, name)
-			fmt.Printf("%v. %v%v\n", i, config.OrgaName, repo["name"])
```

Comment:
The code is not relevant in any way. If you remove debug prints, please do it properly

Reply:
Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)